### PR TITLE
Port over mbed-edge-core-devmode patches to mbed-edge-core

### PR DESCRIPTION
**Pull request by mray190**
There were a bunch of patches in the devmode of edge-core
that were never ported over to the production package of edge-core

Move all the files, and update the series and install scripts
to match

### CODE REVIEW

**Comment by mray190** [comment]
Waiting for the other PR to get merged before merging this PR

**Review by costanic — APPROVED** [review]

**Review by CristianPrundeanuARM — APPROVED** [review]

**Comment by ryannowarm** [comment]
The CI is failing with a known issue with the CI tests themselves: https://go.arm.com/gpn0AZG

**Pull request merged**
